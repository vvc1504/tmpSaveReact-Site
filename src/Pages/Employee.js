import React from 'react'
import{ Trans } from 'react-i18next';

function Employee() {
    return (
        <div className='employee-page'>
            <h1><Trans>Employee</Trans></h1>
        </div>
    )
}

export default Employee
