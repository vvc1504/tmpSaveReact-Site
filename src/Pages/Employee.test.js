import React from 'react';
import renderer from 'react-test-renderer';
import Employee from './Employee';


test('employee snapshort test',()=>{
    const components=renderer.create(
        <Employee/>
    );
    let employeetree=components.toJSON();

    expect(employeetree).toMatchSnapshot();
})