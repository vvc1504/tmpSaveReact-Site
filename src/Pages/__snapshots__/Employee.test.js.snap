// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`employee snapshort test 1`] = `
<div
  className="employee-page"
>
  <h1>
    Employee
  </h1>
</div>
`;
